File: Easy/Staircase/src/main/java/MyCode.java
package main.java;

public class MyCode {
    public static void staircase(int n) {
        String symbol = "#";
        String space = " ";

        for (int i = 1; i<= n; i++){
                for (int j = 1; j<=n-i;j++){
                    System.out.print(" ");
                }
                System.out.printf(symbol.repeat(i));

                System.out.println();

        }

    }
}

File: Easy/Staircase/src/main/kotlin/MyCode.kt
package main.kotlin

fun staircase(n: Int): Unit {
    val symbol: String = "#"
    for (i in 1 until n){
        for (j in 1 until n-i){
         print(" ")
        }
        print(symbol.repeat(i))
        println()
    }

}

fun main(args: Array<String>) {
    val n = readLine()!!.trim().toInt()

    staircase(n)
}